generator/vulkan/render.zig

```diff
@@ -954,10 +1033,10 @@ fn Renderer(comptime WriterType: type) type {
             try self.writer.print(
                 \\pub fn load({s}) !Self {{
                 \\    var self: Self = undefined;
-                \\    inline for (std.meta.fields(Self)) |field| {{
+                \\    inline for (std.meta.fields(Dispatch)) |field| {{
                 \\        const name = @ptrCast([*:0]const u8, field.name ++ "\x00");
-                \\        const cmd_ptr = loader({s}name) orelse return error.InvalidCommand;
-                \\        @field(self, field.name) = @ptrCast(field.field_type, cmd_ptr);
+                \\        const cmd_ptr = loader({s}name) orelse return error.InvalidLoader;
```

Comment:
We now guarantee a command passed in is valid, so the only issue could be with the loader.

Reply:
This is not exactly the case, as the user can still put in a command that is not declared to be loadable. For example, when one tries to load `vkAcquireNextImageKHR` without declaring `VK_KHR_swapchain`.